docs/osdu/integration-test-data/user_info_1.json

```diff
@@ -32,7 +34,9 @@
         "service.search.user",
         "data.default.viewers",
         "data.default.owners",
-        "service.entitlements.admin"
+        "service.entitlements.admin",
+        "service.schema-service.viewers",
+        "service.schema-service.editors"
```

Comment:
Any naming convention we should follow? Because for other we are following with the service suffix

Reply:
These are currently the names hard coded in <service>-core. There is an ADR addressing the same; i.e. keeping the naming consistent; but these are the names we have currently :) 
ADR:
https://community.opengroup.org/osdu/platform/system/schema-service/-/issues/19